aruco_markers_detect_config: 
    config:
          camera_num :  3
          camera_calibration_file_path :  "/home/flfjepl/catkin_ws_vision_location/src/aruco_markers_detect/config/camera_params/"
          camera_calibration_file_names :  ["hikcam0.yaml" , "hikcam1.yaml" ,"hikcam2.yaml"]

          map_file_path :   "/home/flfjepl/catkin_ws_vision_location/src/aruco_markers_detect/config/map/"
          map_file_name :  "markers_map_depu_sfloor.yaml"

          markers_classify_file_path :  "/home/flfjepl/catkin_ws_vision_location/src/aruco_markers_detect/config/markers_classify/"
          markers_classify_file_name :   "markers_classify_depu_sfloor.yaml" 
          topic_cameral_control :  "/camera_control"

          camera_topic_name :  ["/camera0/image_raw", "/camera1/image_raw", "/camera2/image_raw"]   #二维码检测输入图像话题名
          camera_detect_topic_name:  ["/camera0/out"  , "/camera1/out"  ,"/camera2/out"] #二维码检测结果话题  调试 可视化用
          markers_map_pose_topic:  ["/camera0/arucoMarkersMapPose","/camera1/arucoMarkersMapPose","/camera2/arucoMarkersMapPose"]       #输出结果topic名称
          localization_enable:  [true,false,false]
          debug:   true
          markerMapSliceSize:  12.0

